### PR TITLE
chore: use native node13 full icu

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Uses node 13 and drops our FULL_ICU implementation.

The test I changed has the same behaviour when I run it in Chrome.

Screen Shot 2019-10-25 at 1 53 19 PM

Firefox has different behaviour
Screen Shot 2019-10-25 at 1 57 54 PM

I don't think it really matters though.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ aliases:
 executors:
   node_10:
     docker:
-      - image: circleci/node:10.16
+      - image: circleci/node:13.0.1
```
Comment (emmenko) [inline .circleci/config.yml]:
```suggestion
      - image: circleci/node:13.0
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ commands:
   unit_test:
     description: 'Running unit tests'
     steps:
-      - run: yarn run cross-env NODE_ICU_DATA=vendors/node_modules/full-icu jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
```
Comment (emmenko) [inline .circleci/config.yml]: 🎉

File: src/components/inputs/money-input/money-input.story.js [hunk 3]
```diff
@@ -47,6 +47,17 @@ class MoneyInputStory extends React.Component {
   }
 
   render() {
+    const blah = MoneyInput.parseMoneyValue(
```
Comment (tdeekens) [inline src/components/inputs/money-input/money-input.story.js]: Thanks.

Comment (montezume) [reply src/components/inputs/money-input/money-input.story.js]: ahhh

**Comment by montezume** [comment]
Thanks @emmenko for letting us know about this 🙃

**Pull request merged**